src/codegen/codegen_helper_visitor.cpp

```diff
@@ -853,5 +855,62 @@ void CodegenHelperVisitor::visit_after_block(const ast::AfterBlock& node) {
     info.before_after_blocks.push_back(&node);
 }
 
+std::shared_ptr<ast::Compartment> find_compartment(const ast::LongitudinalDiffusionBlock& node,
```

Comment:
Is it used anywhere outside the translation unit? If not, mark `static`, if yes, add docstring.

Reply:
ty